### ISSUE TITLE
Mixed domains results and invalid links to pages in "Most popular content"

### ISSUE THREAD

**Issue by unster** [google/site-kit-wp#554]
## Bug Description
There is incorrect data in "Most popular content" widget on Site Kit Dashboard and Wordpress Admin main page. The data source is Google Analytics.

There are 3 issues:
- I have 2 domains on one Google Analytics page. During the configuration phase I've selected one of the domains that I would like to present the data. It seems that this is ignored and "Most popular content" is based on overall results from both domains on this one account,
- links to those pages are generated based on page title not page url
-- links to pages are pasted after /wp-admin/ part not domain core

## Steps to reproduce
- have two domains setup on Google Analytics
- proceed with typical Site Kit installation

Possible impacts:
- Wordpress is installed in separate sub-folder - [Giving WordPress Its Own Directory](https://wordpress.org/support/article/giving-wordpress-its-own-directory/)
- Yoast SEO plugin is installed

## Screenshots
https://ibb.co/PDk33D6 - items listed in "Most popular" are from "NUNU BABA" site which is my other domain, not core one that I would like to present data,

https://ibb.co/8ByZK0n - as you can see, two things here - "address" is places after /wp-admin/ part not core-domain, and link is based on site title not URL

## Additional Context
PHP 7.3.3
Wordpress: 5.2.3

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation Brief

* 

## Changelog entry

*

**Comment by jamesozzie** [comment]
@unster When you mention you have 2 domains setup within Google Analytics do you have different views setup or have you setup the domains as individual properties under the same account? If not already done I would suggest setting up a view from within Analytics to report on one domain only, before starting Site Kit setup again. 

In relation to your reported URL structure it might be worth opening a new issue specifically for that topic. If you want to provide some more context and include the permalinks setup you have along with any redirection rules or additional information (Are you using a network install, multisite setup etc).

**Comment by jamesozzie** [comment]
@unster Have you managed to setup views from within Analytics or configure the views from within Site Kit for the domains mentioned?
